web/sidebars.js

```diff
@@ -121,6 +111,17 @@ module.exports = {
         'advanced/links',
       ],
     },
+    {
```

Comment:
What do you think about moving it above the Advanced section since it's important?

Reply:
I think deployment comes some time after learning about features, so it made sense to me to first have features (data model, auth, advanced, ...), then comes deployment.

Reply:
People might finish their app without digging into the advanced features and deploy it :) 

But I don't have a strong preference here, I'm fine either way 👍